### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by cheempz**

### CODE REVIEW

**Review by tammy-baylis-swi — APPROVED** [review]
Thank you @cheempz !

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,5 @@
 * @solarwindscloud/eng-apm-instrumentation
 
-# placeholder for GH EMU where this will move to new org for public repos
-#* @solarwinds/eng-apm-instrumentation
+# prepare for move to solarwinds
+@solarwinds/eng-pub-apm-instrumentation
```
Comment (raphael-theriault-swi) [inline CODEOWNERS:4]: This won't work, right now it matches a file named `@solarwinds/eng-pub-apm-instrumentation` and doesn't assign it to everyone.
```suggestion
# prepare for move to solarwinds
* @solarwinds/eng-pub-apm-instrumentation
```

**Review by raphael-theriault-swi — CHANGES_REQUESTED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (raphael-theriault-swi) [reply CODEOWNERS:4]: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

**Comment by cheempz** [comment]
Thanks for the catch @raphael-theriault-swi!  One more review please :)

**Comment by tammy-baylis-swi** [comment]
Phew 🙈 , thank you @raphael-theriault-swi !

**Review by raphael-theriault-swi — APPROVED** [review]

**Pull request merged**